pkgs/servers/authelia/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchzip, autoPatchelfHook, ... }:
+
+let
+  pname = "authelia";
```

Comment:
Perhaps this should be `authelia-bin`, in order to allow for an eventual source-based derivation to be named `authelia`?

Reply:
AFAIK `all-packages.nix` is the important part, *maybe* also the `pname`. The directory itself can be renamed at any time without breaking the users.

Reply:
I have added `-bin` to both `pname` and entry in `all-packages`